### PR TITLE
Update grpc-client-rs to use the richer list_all_twins - EI-732

### PR DESCRIPTION
**Pull request by mihai-dinculescu**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [submodule "api"]
 	path = api
 	url = https://github.com/Iotic-Labs/api.git
+	branch = tags/v0.0.7
```
Comment (mihai-dinculescu) [inline .gitmodules:4]: @Evolter you might have to do the same for the TS client.

Comment (Evolter) [reply .gitmodules:4]: Per DM, I think this does not work for tags.

Comment (mihai-dinculescu) [reply .gitmodules]: Indeed. I've updated the README with the proper instructions.

**Review by bkr-iotic — APPROVED** [review]
:+1:

**Pull request merged**